cli/src/vote.rs

```diff
@@ -800,31 +800,49 @@ pub fn process_create_vote_account(
     let fee_payer = config.signers[fee_payer];
     let nonce_authority = config.signers[nonce_authority];
 
+    let is_feature_active = if !sign_only {
+        let feature_address = solana_sdk::feature_set::vote_state_add_vote_latency::id();
+        rpc_client
+            .get_account(&feature_address)
+            .map(|account| {
+                feature::from_account(&account)
+                    .map(|feature| feature.activated_at.is_some())
+                    .unwrap_or(false)
+            })
+            .unwrap_or(false)
+    } else {
+        true
```

Comment:
This means that a transaction created with `solana create-vote-account --sign-only` on edge cli will continue to fail against all current clusters. Do we care?

Reply:
Both good suggestions.

Reply:
I've incorporated the flipped logic into this PR. However, since the hidden arg is probably going to involve a stupid amount of plumbing, I'd like to do that separately, okay?

Reply:
yeah followup is np